runelite-client/src/main/java/net/runelite/client/plugins/chatfilter/ChatFilterConfig.java

```diff
@@ -113,7 +125,7 @@ default boolean filterLogin()
 		keyName = "collapseGameChat",
 		name = "Collapse Game Chat",
 		description = "Collapse duplicate game chat messages into a single line",
-		position = 9
+		position = 8
```

Comment:
Why are these positions changed?

Reply:
iirc my intent was to have the positions reflect their actual order, making it as clear as possible when viewing `ChatFilterConfig`. Previously it went from 7 to 9, so I corrected it.

Reply:
There's no need for positions to be contiguous. Please revert the unneeded position changes to this file.